### ISSUE TITLE
Move template specific features to template repo or somewhere else?

### ISSUE THREAD

**Issue by ivirshup** [scverse/ecosystem-packages#118]
Why does this repo track repositories using the template? I think this is for tracking ecosystem packages, and using the template is neither sufficient or required to be an ecosystem package.

@grst @flying-sheep

**Comment by grst** [comment]
I think ecosystem packages and repos using the template are related, but I don't mind if this is tracked somewhere else. 
I also don't think it's worth the effort changing something.

**Comment by flying-sheep** [comment]
yeah. last hackathon we briefly discussed where to best put that data and agreed on this.

of course if we had some dedicated database somewhere, that would be the right place
